qubes/vm/__init__.py

```diff
@@ -268,10 +268,19 @@ def load_extras(self):
                 for option in node.xpath('./option'):
                     options[option.get('name')] = option.text
 
+                vm = self.app.domains[node.get('backend-domain')]
+                ident = node.get('id')
+                if 'alias' in options:
+                    vm.events_enabled = True
```

Comment:
This is a bad idea - initially events are disabled, because events handler can assume VM object being fully initialized - which isn't the case here yet.